include/uapi/sound/sof/abi.h

```diff
@@ -29,7 +29,7 @@
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
 #define SOF_ABI_MINOR 23
-#define SOF_ABI_PATCH 0
+#define SOF_ABI_PATCH 1
```

Comment:
not sure what exactly the policy says about this, but in principle this commit doesn't change the ABI - the enum values don't change, nothing essential changes, just names